doc/ci_admin_guide/Docker_registry_provisioning.md

```diff
@@ -0,0 +1,152 @@
+# Provisioning Docker registry
+
+## Preparing machine
+
+Spawn a VM using the CentOS / Ubuntu image (the rest of instruction will assume (CentOS 7),
+use DHCP and name `winci-registry`
+
+Log in using ssh and set the hostname in `/etc/hostname`.
+
+Update the system using `yum upgrade` and reboot.
+
+### Networking
+
+Log in using ssh and change the IP configuration to manual, by modifying the following file:
+`/etc/sysconfig/network-scripts/ifcfg-ens192`. Change the line with `BOOTPROTO` to
+
+```ini
+BOOTPROTO=static
+```
+
+and add the following lines:
+
+```ini
+IPADDR=10.84.12.27
```

Comment:
What parts of CI depend on this fixed IP?

Reply:
See the #202. I'll add a note to this document about it.